lib/susedistribution.pm

```diff
@@ -673,7 +673,6 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
-            wait_still_screen 1;    # Wait a bit to avoid false match on 'text-logged-in-$user', if tty has not switched yet
```

Comment:
@ggardet please show me places where this failed for you so that we can fix the real problem and not introduce needless additional wait times. I do not understand how without the `wait_still_screen` we could have false matches because we look for both the correct user and/or the correct tty.

Reply:
Here is one occurrence: https://openqa.opensuse.org/tests/956900#step/rebootmgr/96 which matches a "logged-in as root" needle (but is not the tty6), but next screen https://openqa.opensuse.org/tests/956900#step/rebootmgr/97 shows tty6 login page (as requested initially).
It leads to a failure, just because of the bad timing.